src/js/views/programs/list/programs-all_views.js

```diff
@@ -0,0 +1,66 @@
+import moment from 'moment';
+
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import PreloadRegion from 'js/regions/preload_region';
+
+import 'sass/modules/list-pages.scss';
+import 'sass/modules/table-list.scss';
+import './programs-list.scss';
+
+const EmptyView = View.extend({
+  tagName: 'tr',
+  template: hbs`
+    <td class="table-empty-list">
+      <h2>{{ @intl.programs.list.programsAllViews.emptyView }}</h2>
+    </td>
+  `,
+});
+
+const ItemView = View.extend({
+  className: 'table-list__item',
+  tagName: 'tr',
+  template: hbs`
+    <td class="table-list__cell w-60">{{ name }}</td>
+    <td class="table-list__cell w-20 program--published">
+    {{#if published}}<span class="program--on">{{fas "toggle-on"}}<span class="program--words">{{ @intl.programs.list.programsAllViews.itemView.on }}</span></span>
+    {{else}}{{far "toggle-off"}}<span class="program--words">{{ @intl.programs.list.programsAllViews.itemView.off }}</span>{{/if}}
+    </td>
```

Comment:
thoughts on this mess? Is it fine? Seems like there might be a better way....

Reply:
it's bordering on needing it's own template, but fine for now.